This patch is a solaris specific one which fixes a build issue
related to change in the prototype of a dladm function. This has
not been submitted to the community.

diff -wpruN '--exclude=*.orig' a~/agent/mibgroup/kernel_sunos5.c a/agent/mibgroup/kernel_sunos5.c
--- a~/agent/mibgroup/kernel_sunos5.c	1970-01-01 00:00:00
+++ a/agent/mibgroup/kernel_sunos5.c	1970-01-01 00:00:00
@@ -62,6 +62,7 @@
 
 #include <libdladm.h>
 #include <libdllink.h>
+#include <libnetcfg.h>
 /*-
  * Includes of local application header files 
  */
@@ -1771,7 +1772,12 @@ int link_to_phydev(char *link, char *phy
         dladm_phys_attr_t dpa;
         dladm_handle_t handle;
 
-        if (dladm_open(&handle, NULL) != DLADM_STATUS_OK) {
+#ifdef NETADM_ACTIVE_PROFILE
+        if (dladm_open(&handle, NULL)
+#else
+        if (dladm_open(&handle)
+#endif
+            != DLADM_STATUS_OK) {
             DEBUGMSGTL(("kernel_sunos5","dladm_open failed "
                        "with errno=%d\n",errno));
             return (-1);
